### DISCUSSION TITLE
Active User - Use any kind of event

### DISCUSSION THREAD

**Discussion by andreivreja** [plausible/analytics#538] [Anything Else]
Hey there!

Took a look over the code after seeing that real time visitors are visitors that had a pageview within the last 5 minutes. There's plenty of times when you'd have super long sessions and still want to know how many active visitors you have.

First thing I tried was simply sending a goal event, rather than sending a pageview, but apparently only pageviews mark users as active. While a proper event/implementation for this would be amazing, is there any chance this could get changed so at least users are marked as live visitors if there's been an event sent, other than `pageview`? 

Unless, of course, events aren't actually linked to sessions; haven't checked that. If they are linked to sessions, I can't think of any downside for making it so any event marks a session as active, shouldn't require much code change, and would be pretty helpful.

**Comment by ukutaht** [comment]
Hey @andreivreja. This makes sense to me and it's not difficult to implement

**Reply by andreivreja** [reply]
Great to heart that! Think this would be pushed soon or not really a priority?

**Reply by ukutaht** [reply]
We have a bunch of other priorities at the moment, sorry
